.github/workflows/deploy.yaml

```diff
@@ -1,5 +1,11 @@
 name: Deployments
 on:
+  # Run the deployment only once staging has been prepared.
+  workflow_run:
+    workflows: ["Trusted checks"]
```

Comment:
I got confused by those names, had to read everything again to make sense of it. Can we call them "build" and "dry-activate" so it's clear what they're doing? One check being more "trusted" than the other doesn't convey much meaning.

Reply:
@fricklerhandwerk This is done.